packages/lodestar/src/api/impl/beacon/pool/index.ts

```diff
@@ -139,9 +151,18 @@ export function getBeaconPoolApi({
             );
           } catch (e) {
             errors.push(e);
+            let archivedPath = "";
+            if (e instanceof SyncCommitteeError && e.action === GossipAction.REJECT) {
+              archivedPath = archiveItem(
+                "syncCommittee",
+                ssz.altair.SyncCommitteeMessage.serialize(signature),
+                toHexString(ssz.altair.SyncCommitteeMessage.hashTreeRoot(signature))
+              );
+            }
+
             logger.error(
               `Error on submitPoolSyncCommitteeSignatures [${i}]`,
-              {slot: signature.slot, validatorIndex: signature.validatorIndex},
+              {slot: signature.slot, validatorIndex: signature.validatorIndex, archivedPath},
```

Comment:
Don't add the archivedPath to the error log itself, just log it afterwards

```ts
logger.error(
              `Error on submitPoolSyncCommitteeSignatures [${i}]`,
              {slot: signature.slot, validatorIndex: signature.validatorIndex},
              e
            );

// then
if (e instanceof SyncCommitteeError && e.action === GossipAction.REJECT) {
              const archivedPath = archiveItem(
                "syncCommittee",
                ssz.altair.SyncCommitteeMessage.serialize(signature),
                toHexString(ssz.altair.SyncCommitteeMessage.hashTreeRoot(signature))
              );
              // Log "Invalid SSZ object written to ${archivedPath}"
            }
```
Then you can abstract the archive logic way better, and don't need `let` vars